scripts/dev-test-rc.sh

```diff
@@ -18,6 +18,15 @@ else
 	ENTERPRISE_REPO=${ENTERPRISE_RELEASE_REPO}
 fi
 
+if [[ ${HZ_TEST_VERSION} == *-SNAPSHOT ]]
+then
+    TEST_REPO=${SNAPSHOT_REPO}
+    ENTRERPRISE_TEST_REPO=${ENTERPRISE_SNAPSHOT_REPO}
+else
+    TEST_REPO=${RELEASE_REPO}
+    ENTRERPRISE_TEST_REPO=${ENTERPRISE_RELEASE_REPO}
```

Comment:
nit: misprint in `ENTRERPRISE_TEST_REPO`. Should be `ENTERPRISE_TEST_REPO`

Reply:
Thanks for that, I forget to check these two scripts for this typo initially